exploration/percent-format.md

```diff
@@ -58,15 +58,47 @@ use one of the existing number-formatting functions `:number` and `:integer` wit
 or use the proposed _optional_ function `:unit` with an option `unit=percent`.
 Combinations of these approached might also be used.
 
+### Unit Scaling
+
+There is a difference between _input_ scaling and _output_ scaling in `MeasureFormat`,
+which is the model for the `:unit` function in Unicode MessageFormat.
+
+For example, an input of <3.5, `meter`> with `meter` as the output unit doesn't scale.
+
+If one supplies <0.35 `percent`> as the input and the output unit were `percent`, 
+`MeasureFormat` would format as 0.35%. 
+Just like `meter` ==> `meter` doesn't scale.
+
+However, if one supplies a different input unit, then percent does scale 
+(just like `meter` ==> `foot`). 
```

Comment:
The `:unit` spec text doesn't actually say what happens if the `unit` is defined both in the _operand_ and in an option.

Reply:
This is documenting `MeasureFormat`'s behavior. We should consider whether to extend `:unit`